measuring_ci/artifacts.py

```diff
@@ -47,29 +46,9 @@
             aws_access_key_id=os.environ.get('TASKCLUSTER_S3_ACCESS_KEY'),
             aws_secret_access_key=os.environ.get('TASKCLUSTER_S3_SECRET_KEY'),
         )
-    loop = asyncio.get_event_loop()
-    log.debug('Fetching S3 artifact info for %s', taskid)
-    artifacts = []
-
-    cont_token = None
     prefix = taskid + '/'
-    while True:
-        if cont_token:
-            kwargs = dict(Bucket=bucket_name, Prefix=prefix,
-                          ContinuationToken=cont_token)
-        else:
-            kwargs = dict(Bucket=bucket_name, Prefix=prefix)
 
-        func = partial(s3_client.list_objects_v2, **kwargs)
-        resp = await loop.run_in_executor(None, func)
-        if resp['KeyCount'] == 0:
-            break
-        artifacts.extend(resp['Contents'])
-        if not resp['IsTruncated']:
-            break
-        cont_token = resp['NextContinuationToken']
-
-    return artifacts
+    return list_s3_objects(s3_client, bucket_name, prefix)
```

Comment:
I think you need an await here and below in find_staged_data_files

Reply:
Thanks!